config/locales/en.yml

```diff
@@ -25,6 +25,9 @@ en:
   components:
     attachment:
       opendocument_html: "This file is in an <a href='https://www.gov.uk/guidance/open-document-format-odf-guidance-for-uk-government/overview-of-productivity-software' target=%{target} class='govuk-link'>OpenDocument</a> format"
+      request_format_text: "This file may not be suitable for users of assistive technology."
+      request_format_cta: "Request a different format"
+      request_format_details_html: "If you use assistive technology and need a version of this document in a more accessible format, please email <a href='mailto:%{contact_email}' class='govuk-link'>%{contact_email}</a>. Please tell us what format you need. It will help us if you say what assistive technology you use."
```

Comment:
we should pass in the target attribute here for consistency

Reply:
I'm wondering is we shouldn't default to '_blank' for this one instead of sharing the same target value with the opendocument link

Reply:
I'm not quite sure what you're suggesting in your comment, but since it's a mailto I'm having second thoughts about my comment. So I now think it's fine with no target.

Reply:
target `_blank` has effect if your email address is set to be a web-based email client (e.g. Gmail) so target still makes sense, since you will be directed to the email provider and loose page context if it's not set. Doesn't have an effect for desktop-based clients though.